source/open-telemetry/tests/operation/aggregate.yml

```diff
@@ -0,0 +1,62 @@
+description: operation aggregate
+schemaVersion: '1.26'
+createEntities:
+  - client:
+      id: &client0 client0
+      useMultipleMongoses: false
+      observeTracingMessages:
+        enableCommandPayload: true
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: operation-aggregate
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: &collectionName0 test
+
+tests:
+  - description: aggregation
+    operations:
+      - name: aggregate
+        object: *collection0
+        arguments:
+          pipeline: &pipeline1
```

Comment:
Nit: was this test originally taken from a larger file? I would have expected `&pipeline0` here.